generate_sample_file.py

```diff
@@ -115,7 +115,7 @@ def get_random_ce_type(self):
         return self.CE_TYPES[random.randint(0, len(self.CE_TYPES) - 1)]
 
     @staticmethod
-    def random_is_community_estab():
+    def random_is_community_estab_address_type_or_level():
```

Comment:
Method name is a bit confusing, maybe just `random_1_in_10` would be clearer now it's being used for multiple different reasons?

Reply:
Done